### PR TITLE
[WIP] Run Pgsql on Managed Service

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/pgsql.py [hunk 1]
```diff
@@ -287,7 +287,10 @@ def wait_for_pgbench_status(self, status, timeout=None):
             timeout (int): Time in seconds to wait
 
         """
-        timeout = timeout if timeout else 900
+        if status == constants.STATUS_RUNNING:
+            timeout = 900
+        elif status == constants.STATUS_COMPLETED:
+            timeout = 9000
```
Comment (PrasadDesala) [inline ocs_ci/ocs/pgsql.py:293]: You dont need to have an additional condition for the timeout, you could directly pass the time out value to the function

File: ocs_ci/templates/workloads/pgsql/PGSQL_Benchmark.yaml [hunk 2]
```diff
@@ -11,11 +11,11 @@ spec:
       clients:
         - 20
       threads: 2
-      transactions: 100
+      transactions: 15000
```
Comment (PrasadDesala) [inline ocs_ci/templates/workloads/pgsql/PGSQL_Benchmark.yaml:14]: Any reason for changing the default parms?
This change effects all the pgsql tests. With these changes pgsql runs for a longer duration which can increase the workloads run time. For functional workload tests the current values holds good.

File: tests/e2e/workloads/app/pgsql/test_pgsql.py [hunk 3]
```diff
@@ -34,10 +34,10 @@ def test_sql_workload_simple(self, pgsql):
         This is a basic pgsql workload
         """
         # Deployment postgres
-        pgsql.setup_postgresql(replicas=1)
+        pgsql.setup_postgresql(replicas=3)
 
         # Create pgbench benchmark
-        pgsql.create_pgbench_benchmark(replicas=1, transactions=600)
+        pgsql.create_pgbench_benchmark(replicas=3)
```
Comment (PrasadDesala) [inline tests/e2e/workloads/app/pgsql/test_pgsql.py:40]: Did you test with replicas setting to 3, is it working? AFAIK, there were some issues with replica  >1

Comment (OdedViner) [reply tests/e2e/workloads/app/pgsql/test_pgsql.py:40]: yes.. It is working

**Pull request closed without merging**